src/Evolution/Systems/Cce/AnalyticSolutions/TeukolskyWave.cpp

```diff
@@ -76,45 +75,42 @@ double TeukolskyWave::pulse_profile_coefficient_c(const double time) const
           12.0 * pow<4>(duration_) * square(extraction_radius_) *
               (square(extraction_radius_) +
                2.0 * extraction_radius_ * retarded_time +
-               3.0 * square(retarded_time))) *
-         amplitude_ * exp(-square(retarded_time) / square(duration_)) /
-         (pow<8>(duration_) * pow<5>(extraction_radius_));
+               3.0 * square(retarded_time)));
 }
 
 double TeukolskyWave::dr_pulse_profile_coefficient_a(const double time) const
     noexcept {
   const double retarded_time = time - extraction_radius_;
   return -dt_pulse_profile_coefficient_a(time) -
-         9.0 *
+         9.0 * amplitude_ * exp(-square(retarded_time) / square(duration_)) /
```

Comment:
can you please write out the derivatives in the documentation so we can compare these easily